box3:
    linear_speed: 20
    angular_speed: 40